conversion/conversion.go

```diff
@@ -435,9 +439,23 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 	return fcopy, n, nil
 }
 
+func dialectToTarget(dialect string) string {
+	if dialect == DIALECT_EXPERIMENTAL_POSTGRES {
+		return TARGET_EXPERIMENTAL_POSTGRES
+	}
+	return TARGET_SPANNER
```

Comment:
Defaulting to spanner even in case of dialect = `DATABASE_DIALECT_UNSPECIFIED` because thats what spanner does to